build.sbt

```diff
@@ -24,12 +21,25 @@ val sonatypeReleaseSettings = Seq(
   )
 )
 
+def artifactProducingSettings(supportScala3: Boolean) = Seq(
+  organization := "com.gu",
+  licenses := Seq(License.Apache2),
+  resolvers ++= Resolver.sonatypeOssRepos("releases"),
+  scalaVersion := "2.13.14",
+  crossScalaVersions := Seq(scalaVersion.value) ++ (if (supportScala3) Seq("3.3.3") else Seq.empty),
+  scalacOptions := Seq(
+    "-release:8",
```

Comment:
`release:11` has become `release:8`?

Reply:
Yep, I'm afraid so- guardian/story-packages is still running on Java 8:

https://github.com/guardian/story-packages/blob/041d6946a1da3d24f687dd4ff643798d8d160887/riff-raff.yaml#L17